knowledge-base/how-to-set-the-report-property-of-the-webreportdesigner-dynamically-through-model.md

```diff
@@ -0,0 +1,106 @@
+---
+title: How to set the Report Property of the WebReportDesigner Dynamically Through Model
+description: Explains how to select the report that will be edited in the web report designer from a view or query it in the URL
+type: how-to
+page_title: Setting the Report Property Dynamically in the WebReportDesginer in .NET Core MVC
+slug: how-to-set-the-report-property-of-the-webreportdesigner-dynamically-through-model
+position: 
+tags: WebReportDesigner, Report
+ticketid: 1543386
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+		<tr>
+			<td>Project Type</td>
+			<td>.NET 5.0 MVC</td>
+		</tr>
+		<tr>
+			<td>Report Designer</td>
+			<td>Web Report Designer</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+This article exaplains how to set the report property of the [Web Report Designer](./web-report-designer) dinamically through a model. By using this approach, you can select the report 
+from one view and pass it to the view which contains the Web Report Designer. You can also query the report's name in the URL of the designer.
+
+## Solution
+1. Add the **DesignerModel**:
+
+```CSharp
+public class DesignerModel
+{
+    private string reportName;
+    public string ReportName { get; set; }
+
+    public DesignerModel(string reportName)
+    {
+        this.ReportName = reportName;
+    }
+}
+```
+
+2. Add a new controller (for example named **ReportingController**). It contains the action that will bring the user to the view of the designer and with the passed the report's name:
```

Comment:
Consider changing to smth like:
" Add a new controller (for example named **ReportingController**). Add the action that will bring the user to the view with the designer and pass the report's name:"